src/components/UserSelector.tsx

```diff
@@ -73,10 +63,10 @@
               key={user.id}
               href={`#user-${user.id}`}
               onClick={() => {
-                onChange(user);
+                dispatch(authorActions.setAuthor(user));
               }}
```

Comment:
The anchor tags are used for user selection, but they are not meant for navigation. Consider using buttons or other interactive elements to avoid misleading semantics.